python/ray/train/lightning/lightning_trainer.py

```diff
@@ -4,7 +4,16 @@
 from typing import Any, Dict, Optional, Type
 import pytorch_lightning as pl
 from pytorch_lightning.plugins.environments import ClusterEnvironment
-from pytorch_lightning.callbacks.progress.base import ProgressBarBase
+
+try:
+    from packaging.version import Version
+except ImportError:
```

Comment:
Add a comment for why this is needed?

Reply:
For ImportError?

Reply:
if packaging version is not a dep of Ray Train, maybe you can just do:
```
try:
    from pytorch_lightning.callbacks.progress import ProgressBar as ProgressBarBase
except ImportError:
    from pytorch_lightning.callbacks.progress.base import ProgressBarBase
```
and comment about it.

Reply:
per offline discussion, versioning is quite useful here

Reply:
Yeah, I think this is a quite succinct solution. But in the long term, we need to fully differentiate import path between 1.x and 2.x, since they are using different import path `import pytorch_lightning` v.s. `import lightning.pytorch`. So I think it could be better to differentiate the import path according to the version number.